src/types.ts

```diff
@@ -1649,14 +1649,26 @@ export type QueryFilter<ObjectType = string> = NonNullable<ObjectType> extends s
       $in?: PrimitiveFilter<ObjectType>[];
       $lt?: PrimitiveFilter<ObjectType>;
       $lte?: PrimitiveFilter<ObjectType>;
+      /**
+       * @deprecated handle the filter explicitly in the code. This is done for performance reasons.
+       */
       $ne?: PrimitiveFilter<ObjectType>;
+      /**
+       * @deprecated handle the filter explicitly in the code. This is done for performance reasons.
```

Comment:
Can you rephrase this line to something that gives more clarity?

`deprecated and will be removed in a future release. Filtering shall be applied client-side.`

Reply:
Done